### PR TITLE
[zh] Sync contribution guidelines (3)

### PR DESCRIPTION
**Pull request by tengqm**
This PR is about the changes to reference generation.

### CODE REVIEW

File: content/zh/docs/contribute/style/style-guide.md [hunk 1]
```diff
@@ -0,0 +1,1207 @@
...
+
+To specify the Kubernetes version for a task or tutorial page, include `min-kubernetes-server-version` in the front matter of the page.
+-->
+### 为 Kubernetes 示例给出版本
+
+代码示例或者配置示例如果包含版本信息，应该与对应的文字描述一致。
+
+如果所给的信息是特定于具体版本的，需要在
+[任务模版](/docs/contribute/style/page-content-types/#task)
+或[教程模版](/docs/contribute/style/page-content-types/#tutorial)
```
Comment (SataQiu) [inline content/zh/docs/contribute/style/style-guide.md:439]:
```suggestion
或 [教程模版](/docs/contribute/style/page-content-types/#tutorial)
```

File: content/zh/docs/contribute/style/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,1207 @@
...
+### Address the reader as "you"
+
+{{< table caption = "Do and Don't - Addressing the reader" >}}
+Do | Don't
+:--| :-----
+You can create a Deployment by ... | We'll create a Deployment by ...
+In the preceding output, you can see... | In the preceding output, we can see ...
+{{< /table >}}  
+-->
+### 将读者成为“你”
```
Comment (SataQiu) [inline content/zh/docs/contribute/style/style-guide.md:1079]:
```suggestion
### 将读者称为”你“
```

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 3]
```diff
@@ -134,31 +122,29 @@ Depending on your page type, put your new file in a subdirectory of one of these
...
 documentation source. The top-level directories under `/content/en/docs/` create
 top-level navigation, and subdirectories each have entries in the table of
 contents.
 -->
+## 将主题放在目录中
 
-目录是使用文档源的目录结构动态构建的。`/content/en/docs/` 下的顶层目录创建顶层导航，它和子目录在目录中都有条目。
+目录是使用文档源的目录结构动态构建的。
+`/content/en/docs/` 下的顶层目录用于创建顶层导航条目，
+这些目录它和他们的子目录在网站目录中都有对应条目。
```
Comment (SataQiu) [inline content/zh/docs/contribute/style/write-new-topic.md:139]:
```suggestion
这些目录和它们的子目录在网站目录中都有对应条目。
```

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 4]
```diff
@@ -202,42 +188,51 @@ following cases (not an exhaustive list):
...
 - 代码显示来自命令的输出，例如 `kubectl get deploy mydeployment -o json | jq '.status'`。
-- 代码不够通用，用户无法验证。例如，你可以嵌入 YAML 文件来创建一个依赖于特定 [FlexVolume](/docs/concepts/storage/volumes#flexvolume)实现的 Pod。
-- 该代码是一个不完整的示例，因为它的目的是高亮显示大文件的部分内容。例如，在描述自定义 [PodSecurityPolicy](/docs/tasks/administer-cluster/sysctl-cluster/#podsecuritypolicy)的方法时，出于某些原因，你可以直接在主题文件中提供一个简短的片段。
-- 由于其他原因，该代码不适合用户验证。例如，当使用 `kubectl edit` 命令描述如何将新属性添加到资源时，你可以提供一个仅包含要添加的属性的简短示例。
+- 代码不够通用，用户无法验证。例如，你可以嵌入 YAML 文件来创建一个依赖于特定
+  [FlexVolume](/docs/concepts/storage/volumes#flexvolume) 实现的 Pod。
+- 该代码是一个不完整的示例，因为其目的是突出展现某个大文件中的部分内容。
+  例如，在描述出于某些原因定制
+  [PodSecurityPolicy](/docs/tasks/administer-cluster/sysctl-cluster/#podsecuritypolicy)
+  的方法时，你可以主题文件中直接提供一个短的代码段。
```
Comment (SataQiu) [inline content/zh/docs/contribute/style/write-new-topic.md:196]:
```suggestion
  的方法时，你可以在主题文件中直接提供一个短的代码段。
```

File: content/zh/docs/contribute/style/style-guide.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/contribute/style/style-guide.md:439]: this space now can be handled properly by the renderer engine.

**Comment by tengqm** [comment]
@SataQiu All feedbacks reviewed and addressed. Thanks!

**Pull request merged**